back/spec/acceptance/files/files_spec.rb

```diff
@@ -176,12 +177,10 @@
     with_options scope: :file do
       parameter :name, 'The name of the file', required: true
       parameter :content, 'The content of the file, encoded in Base64', required: true
-      parameter :project, 'The project to which the file will be uploaded', required: false
-      parameter :description_multiloc, 'The description of the file, as a multiloc string', required: false
-
-      parameter :category, <<~CATEGORY_DESC.squish, required: false
-        The category of the file (values: #{Files::File.categories.values.join(', ')})
-      CATEGORY_DESC
+      parameter :project, 'The project to which the file will be uploaded'
```

Comment:
Is there a reason to leave out `required:` information from the API docs? I feel like for posting it can make sense to document, for updates less as usually all attributes are optional.